cg/cli/workflow/rnafusion/base.py

```diff
@@ -60,23 +48,17 @@ def rnafusion(context: click.Context) -> None:
 @OPTION_STRANDEDNESS
 @DRY_RUN
 @click.pass_obj
-def config_case(
-    context: CGConfig,
-    case_id: str,
-    strandedness: str,
-    dry_run: bool,
-) -> None:
+def config_case(context: CGConfig, case_id: str, strandedness: str, dry_run: bool) -> None:
     """Create samplesheet file for RNAFUSION analysis for a given CASE_ID."""
-
-    analysis_api: AnalysisAPI = context.meta_apis["analysis_api"]
-
+    analysis_api: RnafusionAnalysisAPI = context.meta_apis["analysis_api"]
     LOG.info(f"Creating samplesheet file for {case_id}.")
```

Comment:
```suggestion
    LOG.info(f"Creating sample sheet file for {case_id}.")
```

Reply:
Sure! I can change that. I do prefer samplesheet written together as they also do on nextflow. But I see that in cg is written both ways. I can adapt to this one if this  is the preferred  way.